app/views/rooms/show.html.erb

```diff
@@ -50,6 +50,9 @@
           <dd class="mb-0 pb-2 ps-3">
             <span><%= t("boolean.#{@room.display_on_home_page}") %></span>
           </dd>
+
+          <dt class="pb-2"><%= Room.human_attribute_name(:status) %></dt>
+          <dd class="mb-0 pb-2 ps-3"><%= Room.human_attribute_name("status.#{@room.status}") %></dd>
```

Comment:
Maybe we could use a badge or label? And use decorator to apply color type in function of value